index.html

```diff
@@ -988,6 +988,10 @@
         </p>
         <p>Special thanks to those that helped pioneer the first edition of this model, including especially: Charu Chandiram, Peter Freiling, Nathan Furtwangler, Thomas Olsen, Matt Rakow, Ramu Ramanathan, Justin Rogers, Jacob Rossi, Reed Townsend and Steve Wright.</p>
     </section>
+    <section class="appendix">
+        <h1>Security and Privacy Considerations</h1>
+        <p>There are no known security or privacy impacts resulting from the features outlined in this specification.</p>
```

Comment:
I'll note that: 
> Does not expose personally-identifiable information. 

It might be possible to identify a user by the way they use an input device - as they may generate a particular usage signature over time.